frontend/src/App.tsx

```diff
@@ -1,15 +1,18 @@
 import './App.css'
 import {useEffect, useState} from "react";
-import {Vaccine} from "./models/Vaccine.tsx";
+import {Vaccine, VaccineWithoutId} from "./models/Vaccine.tsx";
 import axios from "axios";
 import Header from "./components/Header.tsx";
 import VaccineList from "./components/VaccineList.tsx";
+import {Route, Routes} from "react-router-dom";
+import Form from "./components/Form.tsx";
 
 export default function App() {
 
     const [vaccines, setVaccines] = useState<Vaccine[]>([]);
 
     useEffect(getAllVaccines, [])
```

Comment:
I think this may need callback function like this  

`
useEffect(( ) => getAllVaccines(), [ ])
`